### DISCUSSION TITLE
Orange PL Profile

### DISCUSSION THREAD

**Discussion by gkorpok** [fabianishere/udm-iptv#298] [Ideas]
Hello, I have a problem with IPTV in Orange Polska. The configuration should look like this: vlan 35 (internet), vlan 839 (iptv), vlan 838 (iptv wide on request) on WAN. Vlan 35 should be directed to all ports. However, two additional vlans should point to a specific LAN port 5 in Udm. Can anyone help me build such a configuration?

**Comment by koper89** [comment]
Hey @gkorpok did you mange it make it work by any chance?

**Reply by mpapierski** [reply]
Did you manage to make it work?

**Reply by had-kel** [reply]
Hi Did you manage to make it work?

**Reply by mpapierski** [reply]
@had-kel My workaround was to put funbox in the network (aka "double nat") instead of ONT and connect patchcord from one extra funbox rj45 port, to a unifi switch, tag it with "IPTV" vlan, set the vlan as external gateway (so it should pick up IP address from funbox DHCP). IPTV  works, and you still can see STB in the dashboard on unifi. 

My understanding is that even if somehow unifi will gain ability to "untie" multiple vlans from a WAN, there are still configuration details still needed to make one of their STBs to work.
